common/components/controllers/LogInController.jsx

```diff
@@ -116,24 +116,28 @@ class LogInController extends React.Component<Props, State> {
                 Sign In
               </Button>
 
-              <Button
-                variant="link"
-                className="LogInController-forgotPassword"
-                onClick={url.navigateToSection.bind(
-                  this,
-                  Section.ResetPassword
-                )}
-              >
-                Forgot Password?
-              </Button>
+              {/* Need to be converted to link to be able to change the header metadata */}
+              <a href={url.section(Section.ResetPassword)}>
+                <Button
+                  variant="link"
+                  className="LogInController-forgotPassword"
+                  >
+                  Forgot Password?
+                </Button>
+              </a>
 
-              <Button
-                variant="link"
-                className="LogInController-createAccount"
-                onClick={url.navigateToSection.bind(this, Section.SignUp)}
+              {/* Need to be converted to link to be able to change the header metadata */}
+              <a
+                type="button"
```

Comment:
Do we need `type="button"` on this anchor element? As I understand it, anchor type accepts MIME types, not input types, so we don't. But please double check me.